samples/msal-node-samples/ElectronSystemBrowserTestApp/src/AuthProvider.ts

```diff
@@ -46,61 +33,60 @@ export default class AuthProvider {
                 },
             },
         });
-
-        this.account = null;
-        this.setRequestObjects();
-    }
-
-    public get currentAccount(): AccountInfo {
-        return this.account;
     }
 
-    /**
-     * Initialize request objects used by this AuthModule.
-     */
-    private setRequestObjects(): void {
-        const baseSilentRequest = {
+    async login(): Promise<AccountInfo> {
+        const tokenRequest: SilentFlowRequest = {
+            scopes: this.authConfig.resourceApi.scopes,
             account: null,
-            forceRefresh: false,
-        };
-
-        this.authCodeUrlParams = this.authConfig.request.authCodeUrlParameters;
-
-        this.authCodeRequest = {
-            ...this.authConfig.request.authCodeRequest,
-            code: null,
-        };
-
-        this.silentProfileRequest = {
-            ...baseSilentRequest,
-            scopes: ["User.Read"],
-        };
-
-        this.silentMailRequest = {
-            ...baseSilentRequest,
-            scopes: ["Mail.Read"],
         };
+        const authResult = await this.getToken(tokenRequest);
+        return this.handleResponse(authResult);
     }
 
-    async getProfileToken(): Promise<string> {
-        return await this.getToken(this.silentProfileRequest);
+    async logout(): Promise<void> {
+        if (!this.account) return;
```

Comment:
You can drop this, but wanted to raise this regardless:

Do we have a universal style guide for our samples? Should we separate this to:

`if (!thus.account) {
    return;
}`
(can't figure out line indentations, but this should be 3 lines)

Just something to consider.

Reply:
We try adhere to typescript style guide but it helps to have our guide.

Reply:
@Robbie-Microsoft, @sameerag  According to the [typescript style guide](https://github.com/microsoft/TypeScript/wiki/Coding-guidelines#style), you can omit the curly braces, but if we prefer to add them, I can definitely do it.

Reply:
Helps readability to have it on multiple lines

Reply:
Done!